docs/markdown/Precompiled-headers.md

```diff
@@ -51,16 +51,27 @@ Using precompiled headers with GCC and derivatives
 --
 
 Once you have a file to precompile, you can enable the use of pch for
-a give target with a *pch* keyword argument. As an example, here's how
-you would use it with a C binary.
+a give target with a *pch* keyword argument. As an example, let's assume
```

Comment:
I think it should be "a given target ..." and from the example below, the keyword argument appears to be *c_pch*, not *pch*

Reply:
"given target" of course. However, *c_pch* is not the only keyword, *cpp_pch* is also possible. I guess, this is why *pch* was chosen here (together with *a* and not *the*).